modules/ROOT/partials/misc/supported-versions.adoc

```diff
@@ -3,19 +3,20 @@
 
 Supported versions of {productname}:
 
-[cols="^,^,^",options="header"]
+[cols="^,^,^,^",options="header"]
 |===
-|Version |Release Date |End of Support
-|7.0 |2024-03-20 |2025-09-20
-|6.8 |2023-12-06 |2025-06-06
-|6.7 |2023-09-13 |2025-03-13
-|6.6 |2023-07-19 |2025-01-19
-|6.5 |2023-06-21 |2024-12-21
-|6.4 |2023-03-29 |2024-09-29
-|6.3 |2022-12-08 |2024-06-08
-|6.2 |2022-09-26 |2024-03-26
-|6.1 |2022-07-13 |2024-01-13
-|6.0 |2022-04-07 |2023-10-07
+|Version |Release Date |End of Community Support |End of Premium Support
+|7.1 |2024-05-08 |2024-11-08 |2025-11-08
+|7.0 |2024-03-20 |2024-05-08 |2025-09-20
+|6.8 |2023-12-06 |2024-10-31 |2025-06-06
```

Comment:
We need to confirm if OECS for 6.8 is the same as the release date for TinyMCE 7.0.
```suggestion
|7.0 |2024-03-20 |2024-05-08 |2025-09-20
|6.8 |2023-12-06 |2024-03-20 |2025-06-06
```